### ISSUE TITLE
Don't sent request, when update copmonent props

### ISSUE THREAD

**Issue by SVITY** [nhagen/react-intercom#35]
It works fine 5-7 times, when props is updated. After that component does not send request to Intercom, when  props is updated.

Any ideas? How to fix it?

**Comment by morgler** [comment]
I noticed the same issue. I just installed react-intercom and connected my user for Intercom to the user in my redux store:

`<Intercom appID="kh0il1tk" {...this.props.user}/>`
…
`const mapStateToProps = (state) => ({
  user: state.user
})`

User data in my case is often updated rapidly in succession (e.g. because the user moves a slider on a form, which continuously updates the user's attributes). I notice that most of the time I don't see the slider's actual position in Intercom, but rather see some value that the slider had in between.

**Comment by nhagen** [comment]
Closing for now. If this is still a problem for anyone after #73 feel free to reply or create another issue.
